rpcs3/rpcs3qt/gui_application.cpp

```diff
@@ -219,10 +224,15 @@ void gui_application::StopPlaytime()
 		return;
 
 	const QString serial = qstr(Emu.GetTitleID());
-	const qint64 playtime = m_gui_settings->GetPlaytime(serial) + m_timer_playtime.elapsed();
-	m_gui_settings->SetPlaytime(serial, playtime);
-	m_gui_settings->SetLastPlayed(serial, QDate::currentDate().toString("MMMM d yyyy"));
-	m_timer_playtime.invalidate();
+	const auto isPUPFirmware = serial.isEmpty();
```

Comment:
Just return here if serial.isEmpty() and invalidate the timer (which shouldn't run here with the new code anyway).
There is also no need to reference the firmware here, since empty serials shouldn't be tracked anyway.
```suggestion
    if (serial.isEmpty())
    {
        m_timer_playtime.invalidate();
        return;
    }
```